include/deal.II/fe/fe_nedelec.h

```diff
@@ -106,6 +106,13 @@ DEAL_II_NAMESPACE_OPEN
  * QGauss<sub>k+2</sub> on each face and in the interior of the cell (or none
  * for N<sub>1</sub>).
  *
+ * It is important to have enough quadrature points in order to perform the
+ * quadrature with sufficient accuracy. For example
+ * [QGauss<dim>(2 * order + 3)](@ref QGauss) can be used for the quadrature
```

Comment:
That's too much. The general rule is that the number of quadrature points must be `poldegree + 1` in each coordinate direction. So a Nedelec element of order 0 has polynomial degree 1, and would need a 2-point Gauss quadrature. Does that not work in your case?

Also, interesting approach to use `[...](@ref QGauss)`. Does this get rendered as you hoped in the doxygen HTML output?

Reply:
> That's too much. The general rule is that the number of quadrature points must be `poldegree + 1` in each coordinate direction. So a Nedelec element of order 0 has polynomial degree 1, and would need a 2-point Gauss quadrature. Does that not work in your case?

`poldegree + 1`  works when you don't have a singularity, but when you have a singularity you really need lot of quadrature points. This makes me think that it should be possible to write a better quadrature formula for Nedelec. Maybe a a quadrature formula that uses the basis functions of Nedelec?

In #8710 @rosskynch mentioned that he needed the quadrature to be high.

Having many quadrature points is inconvenient because that increases the calculation time in the assemble step. How about we write now in the documentation `2 * order + 3` for singularites (as I wrote) and at the same time I try to figure out if there is better quadrature formula for Nedelec.

> Also, interesting approach to use `[...](@ref QGauss)`. Does this get rendered as you hoped in the doxygen HTML output?

I found it in the [doxygen markdown documentation](http://www.doxygen.nl/manual/markdown.html) :)

Reply:
> `poldegree + 1` works when you don't have a singularity, but when you have a singularity you really need lot of quadrature points. This makes me think that it should be possible to write a better quadrature formula for Nedelec. Maybe a a quadrature formula that uses the basis functions of Nedelec?

But the singularity is not related to the Nedelec elemet but rather to the equation you are trying to solve. Unless your coefficients are non-constant, assembling the system matrix (for Maxwell) should be exact using `QGauss(order+2)`.

Reply:
That is right! I tested it and `QGauss(order+2)` gives the right solution :) It is important not to use `QGauss(order+1)` with `Nedelec`. I modified a little bit the text. I placed the clarification in the constructor, because that is where people are going to look. Let me know if the text is ok or you have suggestions.

Reply:
Ah, nice to know!